### ISSUE TITLE
QR Code Error

### ISSUE THREAD

**Issue by juhucoding** [pinetum/AirUnlock-for-Android#9]
Hey, 

thanks for the nice work. However, I have an issue. 

I have installed the apk through F-Droid on my LineageOS 15.1 android phone and the app on macOS 10.13.3.

However, when starting the app on the mac and scanning the QR Code using the phone, it just says "QR Code Error" and keeps scanning. 

I tried restarting the app on the mac and on the phone, but still the same.

A [logcat](https://github.com/pinetum/AirUnlock-for-Android/files/1748750/logcat_airunlock.txt) of the action.

**Comment by 1951FDG** [comment]
@juhucoding you have to use both apps from source, so both the airunlock-for-android & macOS, and try again

**Comment by juhucoding** [comment]
So I need to compile both of them? Just downloading the versions from gitlab did not work, btw – same error.

**Comment by 1951FDG** [comment]
yeah compile this one from source, and additionally, https://github.com/pinetum/AirUnlock-for-Mac

**Comment by ghost** [comment]
Just a heads-up I had no luck doing this in Q1 2020.

I've tried on several different SGS phones and all of my Apple machines, and I've tried all of the forks of this from 2017 to now. This app just does not function anymore.
